__mocks__/popper.js.js

```diff
@@ -0,0 +1,12 @@
+import PopperJs from 'popper.js';
```

Comment:
does this still work if we move it to `src/__mocks__`? if so, let's do that

Reply:
Interestingly it still works after moving the folder inside `src`, though the comment said it had to be adjacent to `node_modules`.